### ISSUE TITLE
Contents of list

### ISSUE THREAD

**Issue by Gitoffthelawn** [paulgb/BarbBlock#27]
I noticed that https://github.com/CHEF-KOCH/BarbBlock-filter-list currently includes more domains.

Is it just due to lack of free time on your end, or are there other reasons?

**Comment by paulgb** [comment]
Yeah, it's a combination of both. My goal here is not to wage war on Admiral so much as to make a statement about DMCA misuse, so I'm not personally going to spend more time trying to add more sites. If people submit pull requests I will accept them if I can verify them. The caveat here is that I don't support people going out and potentially violating anyone's ToS to get domains (see discussion in #4).

I'm not sure the provenance of what's in the repo you linked -- some of it appears to be directly from the ABP / uBlock lists in this repo which is odd because it seems like the point of that list is to provide ABP / uBlock lists from the filter list in the extension.

The domains that have been added to that list, I'm not sure about. dcma.com for example is not an Admiral domain. I'm not sure what else has been added.

**Comment by Gitoffthelawn** [comment]
I honestly haven't followed this whole deal (ordeal?) very closely.

I also haven't studied your list vs @CHEF-KOCH 's list to understand the differences.  I'll post an issue on his repo to let him explain.

What I want to see (and what I think many people may want to see) is simply a list of malware/ad/tracking hosts that were removed from (or not added to) prominent malware/ad/tracking lists.

**Comment by paulgb** [comment]
Noted, thanks. The issue with admiral is that they only actually tried to have one domain removed that I know of, but since they have hundreds of domains it's easy for them to just work around that. So I've been open to adding other admiral domains as well. But if someone wants to actually comprehensively block them it's going to be an ongoing dedicated effort that is outside the scope of this project.

**Comment by Gitoffthelawn** [comment]
Thanks. I think the general procedure most lists take is to be reactive instead of proactive.  IOW, instead of researching and listing every domain owned by CompanyXYZ, only domains actually used for malware/ads/tracking by that company are listed.  Your philosophy matches this standard quite well.

Although not perfect, this seems to work reasonably well.  And, of course, it's impossible to block every domain owned by a company because it takes about 1 minute to register a new domain.  Plus, the use of surrogate companies to register more domains is trivial.

BTW, besides the initial domain you listed, where are people getting all these other domains from?
